src/components/configurable-columns/__spec__.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ConfigurableColumns } from './configurable-columns';
+import Button from './../button';
+import { DraggableContext } from './../drag-and-drop';
+import Form from './../form';
+import Heading from './../heading';
+
+describe('ConfigurableColumns', () => {
+  let wrapper
+  let onCancel = () => { }
+  let onClick = () => { }
+  let onDrag = () => { }
+  let onSave = () => { }
+
+  describe('children', () => {
+    beforeEach(() => {
+      wrapper = shallow(
+        <ConfigurableColumns
+          onCancel={onCancel}
+          onClick={onClick}
+          onDrag={onDrag}
+          onSave={onSave}
+          title='Foo'
+        >
+          <p className='child-node'>Foo</p>
+        </ConfigurableColumns>
+      );
+    });
+    it('renders child nodes', () => {
+      const childNode = wrapper.find('.child-node');
+      expect(childNode.length).toEqual(1);
+    });
+  });
+
+  describe('onCancel', () => {
+    beforeEach(() => {
+      wrapper = shallow(
+        <ConfigurableColumns
+          onCancel={onCancel}
+          onClick={onClick}
+          onDrag={onDrag}
+          onSave={onSave}
+          title='Foo'
+        />
+      );
+    });
+    it('passes the onCancel prop through to the Form onCancel prop', () => {
+      expect(wrapper.find(Form).props().onCancel).toEqual(onCancel);
```

Comment:
If I switch these round then the spec fails with:
```
Expected Function to equal Function.
```
I presume because even though they are both anonymous functions, they still considered different by Jasmine.